src/components/listview/listview.js

```diff
@@ -72,7 +72,7 @@ define(['itemHelper', 'mediaInfo', 'indicators', 'connectionManager', 'layoutMan
         var apiClient = connectionManager.getApiClient(item.ServerId);
 
         var options = {
-            width: width,
+            maxWidth: width,
```

Comment:
We get 80px for playlist icons and 500px for TV show episodes (in TV layout, 83px and 576px).
```suggestion
            maxWidth: width * 2,
```
But this won't help much - images for episodes are 400-600px.